tests/conftest.py

```diff
@@ -133,3 +158,11 @@ def pytest_configure(config):
         "'us76_approx') skips if the 'absorption_spectrum.us76_approx' files "
         "cannot be found in paths resolved by Eradiate.",
     )
+
+    config.addinivalue_line(
+        "markers",
+        "regression: regression tests that are used in CI "
+        "and test reports. Tests that marked as regression"
+        "tests are only run if the regression marker is"
+        "set explicitly.",
+    )
```

Comment:
This should be moved to a PR dedicated to the addition of the regression testing framework.